shared/tests/utility/math/docs/cart_coords_in_cpp.txt

```diff
@@ -0,0 +1,200 @@
+Eigen::Vector3d(-0.492109181440401e+19, -0.080404385128274e+19, -1.673196359027018e+19),
```

Comment:
What is the purpose of this file?
I'd recommend either making it a csv file or not having it at all.
likewise for the others

Reply:
Would it be worth making a utility for parsing csv files of test data? This will probably be a recurring theme as we develop more test files

Reply:
It might be a good idea, the hardest part of all this is where to put it all. I'm not convinced that the doc folder belongs here with the code in the first place